azure-communication-ui/azure-communication-ui/src/main/res/values-ja/azure_communication_ui_strings.xml

```diff
@@ -71,7 +71,7 @@
     <string name="azure_communication_ui_snack_bar_button_dismiss">却下</string>
     <string name="azure_communication_ui_cal_state_error_call_end">エラーが発生したため、通話から削除されました。</string>
     <string name="azure_communication_ui_snack_bar_text_error_call_join">エラーが発生したため、通話に参加できません。</string>
-
+    <string name="azure_communication_ui_call_state_call_evicted"></string>
```

Comment:
Instead of using call twice, could we use 'azure_communication_ui_call_state_evicted'?

Reply:
fixed